config/webpack.dev.js

```diff
@@ -17,7 +17,10 @@ const config = {
     publicPath: "/"
   },
   resolve: {
-    extensions: [".ts", ".tsx", ".js"]
+    extensions: [".ts", ".tsx", ".js"],
+    alias: {
+      '_': path.resolve(__dirname, '../src')
```

Comment:
As we discussed, remove this for now. Everything else looks good though!

Reply:
Ok, removed